### PR TITLE
docs: add useful tool Antidote and Guide to CSS

### PR DESCRIPTION
**Pull request by rsipakov**
#### What does this PR add
Useful tool Antidote

#### Description of added resource
Guide to CSS colors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +181,7 @@
 | Tania Rascia     | https://taniarascia.com                                            | Modern JavaScript, Node.js, and development     | [@taniarascia](https://twitter.com/taniarascia)     |
 | SWYX     | https://swyx.io/writing                                            | Web Development, React and Tech     | [@swyx](https://twitter.com/swyx)     |
 | Sean C Davis    | https://cobwwweb.com/ | Web Development, JamStack | [@seancdavis29](https://twitter.com/seancdavis29)  |
+| Alvaro Montoro | [The Ultimate Guide to CSS Colors (2020 Edition)](https://dev.to/alvaromontoro/the-ultimate-guide-to-css-colors-2020-edition-1bh1#rgb-rgba) | css, html | [@alvaro_montoro](https://twitter.com/alvaro_montoro)
```
Comment (BolajiAyodeji) [inline README.md:184]: This section is supposed to be for blogs only and not an article, kindly remove it or include the Blog details alone.

**Review by BolajiAyodeji — CHANGES_REQUESTED** [review]
Thanks for the amazing PR!

Kindly check my comment below and revert so I can proceed to merge this.

File: README.md (hunk 1 above)
Comment (rsipakov) [reply README.md]: HI @BolajiAyodeji the link to the blog Alvaro Montoro's is updated.

**Review by BolajiAyodeji — APPROVED** [review]

**Comment by BolajiAyodeji** [comment]
@all-contributors please add @rsipakov for docs

**Pull request merged**